.github/default-labels/global-labels.yml

```diff
@@ -0,0 +1,134 @@
+---
+# This is for now only list of standard Espressif issue/PR labels as a reference.
+# Config files can be combined and projects grouped. For example, all org projects
+#  will have set 'global-labels', then defined subset of projects 'global-labels' + 'component-labels', etc ...
+
+# We plan to integrate it with GitHub action https://github.com/EndBug/label-sync
+#  and sync labels to org projects from one place. This needs to be tested first.
+
+# Properties explanation:
+#  name - The name of the label.
+#  color - The color of the label.
+#  description - [optional] The description of the label.
+#  aliases - [optional] An array containing the "aliases" of the label. If an existing label's name is an alias that label will be edited to match your config: this way you don't loose issues and PRs that have been labeled previously.
+
+# Issue types
+- name: 'Type: Bug'
+  description: 'Bug report'
+  color: '#ee0701'
+  aliases: ['bug']
+
+- name: 'Type: Feature Request'
+  description: 'Feature request'
+  color: '#84b6eb'
+  aliases: []
+
+- name: 'Type: Enhancement'
+  description: 'Issue related to project performance'
+  color: '#adecee'
+  aliases: ['enhancement']
```

Comment:
Labels starting with 'Type: ' indicate that this label represents Jira issue type. Enhancement is not an issue type in Jira. Also this type is synonymous to Improvement type that exists and is used in Jira.

Reply:
And description is the same as for 'Type: Performance'